src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -271,80 +271,20 @@ CHIP_ERROR PumpConfigurationAndControlAttrAccess::Write(const ConcreteDataAttrib
 }
 } // namespace
 
-// SDK Callbacks
-
-template <typename T1, typename T2>
-bool IsFeatureSupported(EndpointId endpoint, EmberAfStatus (*getFn1)(chip::EndpointId endpointId, T1 & value),
-                        EmberAfStatus (*getFn2)(chip::EndpointId endpointId, T2 & value))
+bool PumpHasFeature(EndpointId endpoint, PumpFeature feature)
```

Comment:
normally you should be already in the right namespace IMO

I wish we had use the same HasFeature method among all clusters

Reply:
I'm not sure, since this does not seem to have been put into a class similar to other implementations. Would like to refactor this at some point, but not in this PR 😓 

I used PumpHasFeature since I notice this was also changed for LevelControl (https://github.com/project-chip/connectedhomeip/pull/23820/files#diff-0b846b0cf4124840f4d87631d664fed23f7529ce064d963b5021c003176516a9) 

But I agree, maybe one common function template or similar would make sense at some point?

Reply:
I added the PumpConfigurationAndControl namespace, moved the PCC specific functions into that and renamed it to HasFeature.